build.sbt

```diff
@@ -394,6 +398,7 @@ lazy val dynamic = projectMatrix
   .in(file("modules/dynamic"))
   .dependsOn(core, tests % "test->compile", http4s % "test->compile")
   .settings(
+    Compile / packageSrc / mappings := excludeSmithyFiles((Compile / packageSrc / mappings).value),
```

Comment:
This is not necessary, because there is no manifest file in this module that would make it possible for the smithy files it contains to be loaded via `discoverModels`